pkg/kubelet/util/util_test.go

```diff
@@ -149,3 +151,71 @@ func TestIsUnixDomainSocket(t *testing.T) {
 		assert.Equal(t, result, test.expectSocket, "Unexpected result from IsUnixDomainSocket: %v for %s", result, test.label)
 	}
 }
+
+func TestGetContainerByIndex(t *testing.T) {
+	type res struct {
+		container v1.Container
+		ok        bool
+	}
+	tests := []struct {
+		title      string
+		containers []v1.Container
+		statuses   []v1.ContainerStatus
+		idx        int
+		res        res
+	}{
+		{
+			title:      "idx is less than zero",
+			containers: []v1.Container{{Name: "container-1"}},
+			statuses:   []v1.ContainerStatus{{Name: "container-1"}},
+			idx:        -1,
+			res: res{
+				container: v1.Container{},
+				ok:        false,
+			},
+		}, {
+			title:      "idx is large then number of containers",
```

Comment:
This and the next title is not actually correct. Number of containers / container statuses is 1 and the idx is 1.